src/main/java/edu/ohsu/cmp/coach/model/GoalModel.java

```diff
@@ -153,7 +157,7 @@ public int compareTo(@NotNull GoalModel o) {
             Date d1 = getCreatedDate();
             Date d2 = o.getCreatedDate();
             if      (d1 == null && d2 == null)  return 0;
-            else if (d1 != null && d2 != null)  return d1.compareTo(d2);
+            else if (d1 != null && d2 != null)  return d2.compareTo(d1);
```

Comment:
This comparison was incorrect, so the earliest active goal in the EHR was being returned instead of the most recent.